R/helpers.R

```diff
@@ -164,3 +164,40 @@ mergeCountMatrices <- function(cms, transposed=FALSE, ...) {
   }
   return(res)
 }
+
+#' Check for installed package(s)
+#'
+#' @description Check whether a package is installed and suggest how to install from either CRAN or Bioconductor
+#' @param pkgs character Package name(s)
+#' @param details character Helper text (default = "to run this function")
+#' @param install.help character Additional information on how to install package (default = NULL)
+#' @param bioc logical Package(s) is/are available from Bioconductor (default = FALSE)
+#' @param cran logical Package(s) is/are available from CRAN (default = FALSE)
+#' @examples
+#' checkPackageInstalled("sccore", cran = TRUE)
```

Comment:
This example will probably fail CRAN upon upload....

We could try dontrun as well, but I remember running into issues with this

https://stackoverflow.com/questions/12038160/how-to-not-run-an-example-using-roxygen2

It's probably best just to remove the example (sadly)

Reply:
Could checkPackageInstalled("base") work? Otherwise, let's at least go for dontrun

Reply:
Let's try `dontrun`

If there's a problem, I'll just remove the example